TradeItIosTicketSDK2/TradeItTradingTicketViewController.swift

```diff
@@ -452,54 +459,96 @@ class TradeItTradingTicketViewController: TradeItViewController, UITableViewData
         }
 
         cell.selectionStyle = .none
-
-        TradeItThemeConfigurator.configure(view: cell)
+        
+        if let toggle = cell as? TradeItNumericToggleInputCell {
+            TradeItThemeConfigurator.configureNumericToggleInput(toggleInput: toggle)
+        } else {
+            TradeItThemeConfigurator.configure(view: cell)
+        }
```

Comment:
This is a bit eh - is there a way we can annotate this cell in a way that the configurator could read without having to have this switch? In some examples in the theme configurator I've used traits and accessibility settings to do this.